examples/application_commands/slash_command_trees.py

```diff
@@ -0,0 +1,44 @@
+import discord
+from discord import app_commands
+
+client = discord.Client()
+
+# initiate our command tree, with the client
+apple_tree = app_commands.CommandTree(client)
+apples_picked = []
+
+@client.event
+async def on_ready():
+
+    # add the commands to our guild
+    await apple_tree.sync(guild=discord.Object(id=12345))
+
+# initiate the command on top of our coroutine
+@apple_tree.command(guild=discord.Object(id=12345))
+@app_commands.describe(apple='The Apple to Pick') # describe what the option is
+async def pick(interaction: discord.Interaction, apple: str):
+    """Pick an Apple from a tree""" # the docstring representing the commands description
+
+    # make sure the apple isn't already picked
+    if apple in apples_picked:
+        await interaction.response.send_message('This Apple is already picked!')
+
+    # add the apple to the list
+    apples_picked.append(apple)
+    await interaction.response.send_message(f'You picked {apple}!')
+
+# initiate the command on top of our coroutine
+@apple_tree.command(guild=discord.Object(id=12345))
```

Comment:
Instead of 12345 for guild id, maybe using a `GUILD_ID_HERE` placeholder to make it clearer?